ramls/authoritysourcefile.json

```diff
@@ -23,6 +23,14 @@
       "type": "string",
       "description": "Type of authority records stored in source file"
     },
+    "baseUrl": {
+      "type": "string",
+      "description": "Base URL of the source file origin"
+    },
+    "source": {
+      "type": "string",
+      "description": "label indicating where the authority source file entry originates from, i.e. 'folio' or 'local'"
+    },
```

Comment:
Let's introduce the enum. may be: predefined, local

Reply:
folio and local seems to be a common naming for this purpose
Example of settings page with Classification identifier types: ![image](https://user-images.githubusercontent.com/86827432/186876705-e0526fd9-df03-46dc-a570-1d0b4324bede.png)